src/lib_ccx/matroska.c

```diff
@@ -1099,15 +1099,24 @@ char *ass_ssa_sentence_erase_read_order(char *text)
 
 void save_sub_track(struct matroska_ctx *mkv_ctx, struct matroska_sub_track *track)
 {
-	char *filename = generate_filename_from_track(mkv_ctx, track);
-	mprint("\nOutput file: %s", filename);
+	char *filename;
 	int desc;
+
+	if (mkv_ctx->ctx->cc_to_stdout == CCX_TRUE)
+	{
+		desc = 1; // file descriptor of stdout
+	}
+	else
+	{
+		filename = generate_filename_from_track(mkv_ctx, track);
+		mprint("\nOutput file: %s", filename);
 #ifdef WIN32
-	desc = open(filename, O_WRONLY | O_CREAT | O_TRUNC | O_APPEND, S_IREAD | S_IWRITE);
+		desc = open(filename, O_WRONLY | O_CREAT | O_TRUNC | O_APPEND, S_IREAD | S_IWRITE);
```

Comment:
Tabs instead of spaces there

Reply:
I've indeed used a tab there. Here's `vim`'s result of running the `:ascii` command at that character:
![image](https://user-images.githubusercontent.com/66527624/203847096-a06b96e1-db98-4c86-aed8-a2fbfa4d75af.png)
You can also see in the bottom right that `vim` considers the cursor to be at line 1114, character 2, and display width 16 (each tab occupying 8 screen characters): `1114,2-16`.

Reply:
I might have misunderstood you.

Do you mean I should use spaces instead of tabs? The rest of the file (untouched by me) uses tabs.